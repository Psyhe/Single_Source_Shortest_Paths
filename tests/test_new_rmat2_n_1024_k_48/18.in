1024 394 414
394 395 105
394 665 80
394 718 121
394 130 220
394 395 221
394 968 179
394 392 50
394 18 242
394 142 159
394 387 60
394 282 79
394 386 136
394 994 196
394 970 46
394 392 222
394 786 11
394 280 33
394 410 187
394 426 197
394 398 166
394 458 181
395 415 251
395 399 124
395 266 149
395 215 138
395 407 81
395 139 250
395 387 167
395 859 21
395 259 133
395 394 105
395 906 141
395 394 221
395 387 18
395 399 6
396 397 88
396 668 82
396 388 17
396 516 125
396 8 99
396 928 77
396 8 173
396 900 182
396 420 128
396 75 149
396 908 22
396 206 247
396 276 200
396 391 24
396 397 166
396 136 59
396 902 21
397 396 166
397 411 104
397 813 75
397 385 37
397 396 88
397 925 124
397 153 118
397 911 191
397 269 211
397 399 78
397 265 225
398 182 165
398 414 224
398 430 152
398 267 198
398 408 55
398 910 215
398 394 166
398 474 68
398 388 109
398 171 247
398 461 91
398 262 147
398 294 250
398 142 111
399 965 108
399 666 170
399 407 197
399 909 62
399 397 78
399 940 142
399 415 6
399 395 124
399 911 217
399 135 0
399 395 6
400 388 209
400 180 179
400 912 183
400 144 60
400 160 85
400 404 161
400 913 148
400 312 121
400 176 90
400 793 61
400 946 254
400 900 33
400 786 5
400 336 153
400 499 240
400 432 78
400 336 210
400 432 56
400 409 230
400 464 89
400 432 62
400 384 56
400 402 116
401 435 223
401 308 221
401 16 125
401 449 117
401 388 214
401 913 231
401 403 76
401 385 241
401 279 59
401 193 36
401 413 120
401 145 169
401 384 53
401 145 160
401 417 245
401 24 48
401 408 242
402 434 232
402 387 120
402 386 220
402 435 145
402 638 11
402 978 147
402 130 135
402 386 12
402 400 116
402 404 254
402 163 104
402 306 64
402 403 79
402 466 251
402 786 58
402 80 87
402 130 152
403 402 79
403 467 148
403 147 79
403 411 235
403 466 215
403 18 248
403 917 152
403 401 76
403 155 37
403 417 185
404 405 157
404 388 198
404 656 44
404 337 242
404 402 254
404 956 133
404 468 64
404 172 42
404 400 161
404 656 131
404 276 116
404 212 118
404 336 143
404 468 191
404 260 218
404 277 119
404 916 152
404 406 148
404 388 37
404 86 70
404 4 209
405 773 214
405 407 245
405 159 82
405 269 105
405 437 230
405 261 35
405 244 72
405 276 228
405 144 21
405 21 61
405 927 143
405 404 157
406 389 7
406 278 79
406 284 72
406 284 78
406 162 198
406 278 243
406 147 28
406 438 217
406 310 89
406 404 148
406 566 102
406 646 88
406 278 182
406 198 88
406 790 218
406 662 241
407 294 241
407 395 81
407 468 13
407 390 165
407 302 45
407 932 201
407 214 176
407 39 2
407 504 172
407 150 228
407 399 197
407 405 245
407 503 84
407 902 186
408 912 73
408 384 161
408 393 223
408 280 110
408 412 119
408 920 109
408 28 230
408 474 125
408 904 21
408 432 155
408 401 242
408 153 221
408 648 255
408 88 160
408 320 158
408 280 26
408 392 67
408 145 126
408 413 117
408 380 116
408 472 127
408 398 55
408 789 50
408 280 3
409 411 94
409 400 230
409 447 20
409 905 192
409 217 156
409 793 27
409 441 103
409 236 71
409 488 118
409 273 5
409 145 196
409 329 36
409 921 45
410 276 238
410 170 225
410 507 82
410 27 6
410 394 187
410 920 44
410 414 203
410 26 66
410 976 169
410 299 153
410 413 108
410 155 74
410 154 91
411 906 215
411 155 166
411 443 64
411 315 13
411 899 225
411 435 84
411 339 42
411 153 171
411 397 104
411 403 235
411 409 94
411 155 148
412 510 212
412 156 9
412 156 172
412 821 190
412 156 123
412 136 95
412 136 97
412 280 204
412 280 96
412 284 215
412 413 119
412 440 102
412 408 119
412 240 251
413 412 119
413 188 7
413 177 39
413 285 115
413 410 108
413 401 120
413 479 160
413 392 39
413 408 117
414 158 169
414 398 224
414 410 203
