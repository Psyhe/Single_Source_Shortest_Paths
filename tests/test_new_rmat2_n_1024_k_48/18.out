49
37
65
98
90
31
43
63
53
91
74
49
69
51
35
43
72
80
51
60
275
